AutoRunCustomTool64/source.extension.vsixmanifest

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+  <Metadata>
+    <Identity Id="AutoRunCustomTool64.bdbd7370-6a02-43d2-8452-9e6e99d0533e" Version="1.0" Language="en-US" Publisher="Diligent Corporation" />
+    <DisplayName>AutoRunCustomTool64</DisplayName>
+    <Description>Empty VSIX Project.</Description>
+  </Metadata>
+  <Installation>
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[17.0, 18.0)">
+      <ProductArchitecture>amd64</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Version="[17.0,18.0)" Id="Microsoft.VisualStudio.Pro">
+      <ProductArchitecture>amd64</ProductArchitecture>
```

Comment:
I can't find any clear documentation about this, but I suspect it's not actually necessary to mention both Pro and Community. Just Community should be enough.

Reply:
It would seem you are correct. I will remove this additional target.